server/repository/src/db_diesel/stock_on_hand.rs

```diff
@@ -9,7 +9,7 @@ table! {
         item_id -> Text,
         item_name -> Text,
         store_id -> Text,
-        available_stock_on_hand -> BigInt,
```

Comment:
Note, this isn't a problem for Sqlite but luckly Postgres tests were failing otherwise I would have missed this one. This change came indirectly from changing the view.